### PR TITLE
Correct the tabs and newer README

### PR DESCRIPTION
**Pull request by snowc0de**
I changed the readme file into MarkDown and added a new section to explain the installation and the dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
-Controls:
---------
+## Dependencies
+* Python 2 (2.6 or 2.7)
+* Pygame (`pip install pygame`)
+
+## Installation
+1. Run the following commands :
+```
+sudo git clone https://github.com/EricBurnett/GameOfLife.git
```
Comment (EricBurnett) [inline README.md:17]: `git clone` should not require sudo - can that be removed?

**Review by EricBurnett — CHANGES_REQUESTED** [review]
Thanks for the updates! Appreciate it.

File: README.md (hunk 1 above)
Comment (snowc0de) [reply README.md:17]: That depends the area, I often had an issue with that because of being non-root user

Comment (snowc0de) [reply README.md:17]: But it can be removed in fact I tried in another area on my hard drive and it works

Comment (EricBurnett) [reply README.md:17]: Merging as-is - thanks for validating; I can remove the `sudo` post-merge :).

**Pull request merged**